lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -252,11 +252,11 @@ protected void PeriodicUploadLog(object source, ElapsedEventArgs e)
             }
         }
 
-        internal string[] ReadLogFile(string logFileName, string subfolder = "driver", string testFolder = DefaultRuntimeFolder)
+        internal string[] ReadLogFile(string logFileName, string subfolder = "driver", string testFolder = DefaultRuntimeFolder, int retryCount = 60)
```

Comment:
In some tests, sometimes after 60 retry for reading the log file, if the test itself is not completed yet, the test will fail. In this test especially, when we close tasks, TcpConnection could retry for longer time than log file waiting time, passing a bigger number is safer.